auction/src/lib.rs

```diff
@@ -280,6 +280,8 @@ pub mod pallet {
 		CannotListListedNFTs,
 		/// Cannot auction capsules.
 		CannotListCapsulesNFTs,
+		/// Cannot list because the NFT secret is not synced.
+		CannotListNotSyncedSecretNFTs,
```

Comment:
New errors always add at the end of the list. 
Since auctions are not deployed we can safely ignore this rule rn.